pages/windows/if.md

```diff
@@ -0,0 +1,32 @@
+# if
+
+> Performs conditional processing in batch scripts.
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/if>.
+
+- Execute the specified commands if the condition is true:
+
+`if {{condition}} (echo Condition is true)`
```

Comment:
Oh yeah, we're missing some token syntax here too - the same also goes for the other examples.
```suggestion
`if {{condition}} ({{echo Condition is true}})`
```

Reply:
Agreed.

Reply:
But `echo Condition is true` is not a placeholder, it's a concrete command. I've removed this placeholder to simplify tlds page.